cli/src/commands/validate/validate.ts

```diff
@@ -0,0 +1,95 @@
+import Ajv2020 from 'ajv/dist/2020.js';
+import { existsSync, promises as fs, readFileSync, readdirSync, statSync } from 'fs';
+import pkg from '@stoplight/spectral-core';
+const { Spectral } = pkg;
+import { getRuleset } from '@stoplight/spectral-cli/dist/services/linter/utils/getRuleset.js';
+
+export default async function validate(jsonSchemaInstantiationLocation: string, jsonSchemaLocation: string, metaSchemaPath: string) {
+    let exitCode = 0;
+    try {
+        const ajv = new Ajv2020({ strict: false });
+
+        loadMetaSchemas(ajv, metaSchemaPath);
+
+        console.info(`Loading pattern from : ${jsonSchemaLocation}`);
+        const jsonSchema = await getFileFromUrlOrPath(jsonSchemaLocation);
+
+        console.info(`Loading pattern instantiation from : ${jsonSchemaInstantiationLocation}`);
```

Comment:
We should use Winston for logging here. It's already been added as a dependency and there are examples of how to use it in the generate command that Will wrote

Reply:
I am planning to add Winston in a following PR. I think it would be good to extract the logic to instantiate the logger in an external file because we reuse it across the 3 commands